src/SonarScanner.Shim/Resources.resx

```diff
@@ -244,4 +245,7 @@ Possible causes:
   <data name="WARN_TestResultsNotFound" xml:space="preserve">
     <value>Test result file does not exist at the specified location. Path: {0}</value>
   </data>
+  <data name="WARN_EmptyProjectGuids" xml:space="preserve">
+    <value>The following projects don't have any project GUID and will be skipped from the analysis: {0}</value>
```

Comment:
Probably we should mention that the projects are not inside a solution here: `The following projects don't have ProjectGuid and are not part of a solution and will be skipped from analysis: {0}` (or something better)

Reply:
They could be part of a solution that would be somehow malformed.

Reply:
How about the following then?
_"The following project do not have a valid ProjectGuid and were not built using a valid solution (.sln) and will be skipped from analysis..."_

Reply:
@duncanp-sonar is it ok to have 2 `and` in the sentence?